src/main/webapp/resources/js/pages/projects/project-analyses/components/SingleSampleAnalysisOutputs.jsx

```diff
@@ -0,0 +1,182 @@
+          >
+            {sampleName}
+          </a>
+        );
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.file"),
+      key: "fileName",
+      dataIndex: "filePath",
+      render(filePath, record) {
+        let splitItems = filePath.split("/");
+        let fileName = splitItems[splitItems.length - 1];
+        return fileName + " (" + record.analysisOutputFileKey + ")";
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.analysisType"),
+      key: "analysisType",
+      dataIndex: ["analysisType", "type"],
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.pipeline"),
+      key: "pipeline",
+      dataIndex: "workflowDescription",
+      render(workflowDescription) {
+        return workflowDescription.name + " " + workflowDescription.version;
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.analysisSubmission"),
+      key: "analysisSubmissionName",
+      dataIndex: "analysisSubmissionName",
+      render(analysisSubmissionName, record) {
+        return (
+          <a
+            href={setBaseUrl(`/analysis/${record.analysisSubmissionId}`)}
+            target="_blank"
+          >
+            {analysisSubmissionName}
+          </a>
+        );
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.submitter"),
+      key: "submitter",
+      dataIndex: "userFirstName",
+      render(userFirstName, record) {
+        return userFirstName + " " + record.userLastName;
+      },
+    },
+    {
+      ...dateColumnFormat(),
+      title: i18n("SingleSampleAnalysisOutputs.createdDate"),
+      key: "createdDate",
+      dataIndex: "createdDate",
+      sorter: false,
+    },
+  ];
+
+  // Get the selected row ids (AnalysisOutputFile id)
+  let rowSelection;
+  rowSelection = {
+    selectedRowKeys: selected,
+    onChange: (selectedRowKeys) => setSelected(selectedRowKeys),
+    getCheckboxProps: (record) => ({
+      name: record.analysisOutputFileId,
+    }),
+  };
+
+  // Function to filter out outputs by the search term
+  const filterOutputsByTerm = (searchStr) => {
+    if (
+      searchStr.trim() === "" ||
+      searchStr === "undefined" ||
+      searchStr === null
+    ) {
+      setFilteredOutputs(null);
+    } else {
+      searchStr = String(searchStr).toLowerCase();
+      // filePath contains the file name so we use it in the search
+      const outputsContainingSearchValue = outputs.filter(
+        (analysisOutput) =>
+          analysisOutput.sampleName.toLowerCase().includes(searchStr) ||
+          analysisOutput.analysisType.type.toLowerCase().includes(searchStr) ||
+          analysisOutput.filePath.toLowerCase().includes(searchStr) ||
+          analysisOutput.userFirstName.toLowerCase().includes(searchStr) ||
+          analysisOutput.userLastName.toLowerCase().includes(searchStr)
+      );
+      setFilteredOutputs(outputsContainingSearchValue);
+    }
+  };
+
+  return (
+    <>
```

Comment:
Redundant empty tag

Reply:
My error sorry

Reply:
Ok, so I did some playing because I did not like the div here.  Something I always forget is that `Row` can do the justify content for you, and if you are adding a margin, you should use the `Space` component instead for better consistency.  

```javascript
    <Space direction="vertical" style={{ display: "block" }}>
      <Row justify={"space-between"}>
        <Button>
          <IconDownloadFile style={{ marginRight: SPACE_XS }} />
          {i18n("SingleSampleAnalysisOutputs.download")}
        </Button>
        <Search
          style={{ width: 300 }}
          onSearch={filterOutputsByTerm}
          placeholder={i18n("SingleSampleAnalysisOutputs.searchPlaceholder")}
          allowClear
        />
      </Row>
      <Table
        rowKey={(record) => record.analysisOutputFileId}
        loading={isLoading}
        scroll={{ x: "max-content" }}
        columns={columns}
        dataSource={filteredOutputs || (!isLoading && outputs)}
        tableLayout="auto"
        rowSelection={rowSelection}
        pagination={{
          pageSizeOptions: PAGE_SIZE_OPTIONS,
          defaultPageSize: DEFAULT_PAGE_SIZE,
          showSizeChanger: true,
        }}
      />
    </Space>
```

Reply:
Thanks. I can't believe I forgot about the space/row. I've used it before..doh. Updated in ad1cfed0d01bd31b029c25104beb4659dd3c820e